metrics-server/Dockerfile

```diff
@@ -1,27 +1,33 @@
+FROM ubuntu:latest as kubectl
+RUN apt-get update
+RUN apt-get install -y curl
+RUN curl -fsSL https://dl.k8s.io/release/v1.13.4/bin/linux/amd64/kubectl > /usr/bin/kubectl
+RUN chmod a+rx /usr/bin/kubectl
+
 # Build the manager binary
 FROM golang:1.13 as builder
 
-WORKDIR /workspace
+# Copy in the go src
+WORKDIR /go/src/guestbook-operator
 # Copy the Go Modules manifests
 COPY go.mod go.mod
 COPY go.sum go.sum
 # cache deps before building and copying source so that we don't need to re-download as much
 # and so that source changes don't invalidate our downloaded layer
 RUN go mod download
 
-# Copy the go source
+COPY vendor/   vendor/
```

Comment:
The `vendor/` isn't checked into this repo's metric-server directory.
Is this just for local development?
I don't see `vendor/` listed in .gitignore except for an excluded rule about autogen files.

If I'm understanding correctly, would we change the order of `go mod download` and `COPY` so that it can use any local vendored modules if present?
We could also just remove this `COPY` statement.